services/net/syndication/SyndicationAction.cs

```diff
@@ -111,6 +111,7 @@ private async Task ImportFeedAsync(IIngestServiceActionManager manager, Syndicat
                 }
                 else continue;
 
+                reference = await FindContentReferenceAsync(manager.Ingest.Source?.Code, item.Id);
```

Comment:
Why are you fetching the content reference again?  You have a copy from line 93 and 109 when it was updated last.  All this change introduces is a few millisecond hope and a prayer difference.  We need to find out what is causing the concurrency error.  Which service is attempting to update old values?

Reply:
The table is content_reference in this case, thanks

Reply:
We are only running a single Syndication Service presently.  I believe it is single threaded when it is running each ingest.  Which means it shouldn't be competing with anything else.

Reply:
Updated, thanks